tests/config/nextflow.config

```
@@ -14,12 +14,17 @@ if ("$PROFILE" == "singularity") {
     singularity.enabled = true
     singularity.autoMounts = true
 } else if ("$PROFILE" == "conda") {
-    conda.enabled = true
+    params.enable_conda = true
+    conda.enabled=true
```

Restored the initial code as `params.enable_conda` is not used since #2654 (which happened after you started #1706)

```suggestion
    conda.enabled = true
```